Makefile

```diff
@@ -1,21 +1,31 @@
-LIBHDBPP_DIR = .libhdbpp
-LIBHDBPP_INC = ./$(LIBHDBPP_DIR)/src
+PREFIX=/usr/local
 
 DBIMPL_INC = `mysql_config --include`
 DBIMPL_LIB = `mysql_config --libs_r`
 
-TANGO_INC := ${TANGO_DIR}/include/tango
-OMNIORB_INC := ${OMNIORB_DIR}/include
-ZMQ_INC :=  ${ZMQ_DIR}/include
+ifdef TANGO_INC
+	INC_DIR += -I${TANGO_INC}
+endif
 
```

Comment:
missing OMNIORB_INC and ZMQ_INC

Reply:
Yes, I removed them, we do not need them to build this shared library. I have managed to reduce the number of includes/libraries across a numberof the HDB++ modules. 

All these modules build fine. Better, they are packages into debian packages.

Reply:
But it cannot compile in our actual environment: including tango.h requires omniorb and zeromq headers also.

Reply:
Let me double check a few different compile environments and see if it causes a problem. Will add a note back on here tonight/tomorrow.

Reply:
Actually, it was quicker to check than I thought. 

I removed the zmq headers from the include directory and I can still compile the library. The zmq headers are used by eventsupplier/consumer.h, which do not get used.

But you are correct about the omni includes, tango_config.h includes these, so I will add that flag back in (as OMNIORB_INC).

I will double check hdbpp-em/cm also.

Reply:
If I don't include zeromq I have:
g++ -std=gnu++0x -Wall -DRELEASE='"eadURL"' `mysql_config --include` -I/usr/local/tango-9.2.5a/include/tango -I/usr/local/omniorb-4.2.1/include -I../libhdbpp/src -fPIC -c src/LibHdb++MySQL.cpp -o obj/LibHdb++MySQL.o
In file included from /usr/local/tango-9.2.5a/include/tango/event.h:41:0,
                 from /usr/local/tango-9.2.5a/include/tango/tango.h:133,
                 from ../libhdbpp/src/LibHdb++.h:24,
                 from src/LibHdb++MySQL.h:24,
                 from src/LibHdb++MySQL.cpp:23:
/usr/local/tango-9.2.5a/include/tango/zmq.hpp:24:17: fatal error: zmq.h: No such file or directory

Reply:
Hmm, interesting. Better to be safe and add it back then! I am not sure why mine compiled. I will take another look, perhaps we have the header in another location I am not aware of.

Reply:
Ha, we do. We have installed on the base system also.

Good spotting. I will add the flags back so we have full support. 

Just as a note. If you use the debian packages for the dependencies, you can eliminate the flags in your build.